hdkeychain/extendedkey.go

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2014-2016 The btcsuite developers
-// Copyright (c) 2015-2020 The Decred developers
+// Copyright (c) 2015-2021 The Decred developers
```

Comment:
Not that it matters much since they are bumped before release, but I noticed that you bumped the copyright version here and in the tests file, but not in `hdkeychain/doc.go`

Reply:
Good call.  I was planning on tagging this commit since I usually only do release commits if some change is actually needed (like updating the dependencies).